mlir/include/mlir/Dialect/SparseTensor/Transforms/Passes.h

```diff
@@ -40,6 +40,14 @@ enum class SparseParallelizationStrategy {
   kAnyStorageAnyLoop
 };
 
+/// Defines a scope for reinterpret map pass.
+enum class ReinterpretMapScope {
+  kAll,           // reinterprets all applicable operations
+  kGenericOnly,   // reinterprets only linalg.generic
+  kExceptGeneric, // reinterprets operation other than linalg.generic (e.g.,
```

Comment:
can you try to keep this on one line? I like // xxx comments to stay unbroken (if possible)